README.md

```diff
@@ -7,7 +7,7 @@
 [![Codecov](https://img.shields.io/codecov/c/github/datatogether/webapp.svg?style=flat-square)](https://codecov.io/gh/datatogether/webapp)
 [![CI](https://img.shields.io/circleci/project/github/datatogether/webapp.svg?style=flat-square)](https://circleci.com/gh/datatogether/webapp)
 
-**Webapp** is the main public-facing web application to allow user to contribute metadata about content that has been crawled & archived from Data Together. Beta code runs on [archivers.co](https://archivers.co) and production code on [datatogether.org](https://datatogether.org).
+**Webapp** is the main public-facing web application to allow user to contribute metadata about content that has been crawled & archived from Data Together. Beta code runs on [archivers.co](https://archivers.co) and production code on [datatogether.org](https://datatogether.org). Use this repositories to improve the user interface presented in the web application. To add new features to the Data Together platform, you'll have to land those in one or more of the other repositories in the [Data Together Github organization](https://github.com/datatogether/). 
```

Comment:
Eventually we'll want to update the subdomain for production code (And actually I think move our staging over there too?